airflow/providers/google/cloud/triggers/gcs.py

```diff
@@ -97,3 +99,101 @@ async def _object_exists(self, hook: GCSAsyncHook, bucket_name: str, object_name
             if object_response:
                 return "success"
             return "pending"
+
+
+class GCSCheckBlobUpdateTimeTrigger(BaseTrigger):
+    """
+    A trigger that makes an async call to GCS to check whether the object is updated in a bucket.
+
+    :param bucket: google cloud storage bucket name cloud storage where the objects are residing.
+    :param object_name: the file or folder present in the bucket
+    :param ts: datetime object
```

Comment:
missing description of meaning of param

Reply:
added missing description of meaning of param